app/src/main/java/waiter/OutputStreamer/OutputStreamer.java

```diff
@@ -0,0 +1,20 @@
+package waiter.OutputStreamer;
+
+import java.io.PrintStream;
+import java.net.Socket;
+
+import java.io.IOException;
+
+public class OutputStreamer implements Writer {
+
+    private final Socket socket;
+
+    public OutputStreamer(Socket socket) {
+        this.socket = socket;
+    }
+
+    public void writeLine(String toClient) throws IOException {
+        PrintStream printStream = new PrintStream( this.socket.getOutputStream() );
```

Comment:
Is the beginning and trailing spacing here intentional? `... new PrintStream( this.socket.getOutputStream() );`